plugins/ltac/tacintern.ml

```diff
@@ -402,13 +402,13 @@ let dump_glob_red_expr = function
     try
       Dumpglob.add_glob ?loc:r.loc
         (Smartlocate.smart_global r)
-    with e when CErrors.noncritical e -> ()) occs
+    with e when noncritical e -> ()) occs
```

Comment:
Noise diff, keep the qualification here.

Reply:
I changed it for internal consistency of qualification within the file (considering e.g. that `user_err` is not qualified). But that's true that it introduces noise in the diff. Let me try to backtrack on it.